### PR TITLE
Update install_ubuntu_src.md

### PR DESCRIPTION
**Pull request by pxalcantara**
Just a simple point if the person has problems trying to have the program recognized as I had

### CODE REVIEW

**Comment by pxalcantara** [comment]
Friendly ping @maryaB-osr.
Another question @maryaB-osr and @chapulina , I'd like to collaborate with the [contribuiting](https://github.com/ignitionrobotics/docs/blob/master/contributing.md) with more details about some problems that I had in my first PR. Should I separate the PR by topics or, once It's not big modifications, I can do it just in the same PR?

File: citadel/install_ubuntu_src.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ pip install vcstool
 ```bash
 pip install -U colcon-common-extensions
 ```
+Test the programs installed using PIP, if the system is not recognising the commans, and you're using a sistem that is compatible with Debian or Ubuntu packages, **install using apt is a good idea because the system is automatically configured**.
```
Comment (chapulina) [inline citadel/install_ubuntu_src.md:31]: > Test the programs installed using PIP

A user who doesn't know how these programs work wouldn't know how to test them. How about saying something like:

```suggestion
Check that no errors were printed while installing with PIP. If your system is not recognising the commands, and you're using a system that is compatible with Debian or Ubuntu packages, see the instructions below to install using `apt`.  
```

Comment (pxalcantara) [reply citadel/install_ubuntu_src.md:31]: I liked it, better, thanks!

**Comment by pxalcantara** [comment]
What does this error mean? I read but couldn't understand the way to solve it.

**Comment by chapulina** [comment]
> What does this error mean? I read but couldn't understand the way to solve it.

Don't worry about that, it's an issue on our end, ticketed #35 .

**Pull request merged**